source/assets/stylesheets/_navigation.scss

```diff
@@ -6,6 +6,15 @@
   }
 }
 
+.menu-trigger {
+  display: none;
+  text-align: right;
+
+  @media (max-width: 768px) {
```

Comment:
I wonder if this `768px` width is used frequently enough to be extracted to a constant?